gstreamer-mpegts-sys/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[badges.travis-ci]
+branch = "master"
+repository = "sdroege/gstreamer-sys"
+
+[build-dependencies]
+pkg-config = "0.3.7"
+
+[dependencies]
+libc = "0.2"
+
+[dependencies.glib-sys]
+git = "https://github.com/gtk-rs/sys"
+
+[dependencies.gobject-sys]
+git = "https://github.com/gtk-rs/sys"
+
+[dependencies.gstreamer-base-sys]
+path = "../gstreamer-base-sys"
+
+[dependencies.gstreamer-sys]
+path = "../gstreamer-sys"
+
+[features]
+v1_2 = []
+v1_4 = ["v1_2"]
+v1_6 = ["v1_4"]
+v1_8 = ["v1_6"]
+v1_10 = ["v1_8"]
+v1_12 = ["v1_10"]
```

Comment:
All this feature stuff not needed as you have 1.12 as min

Reply:
Fix coming.